### DISCUSSION TITLE
Some WebDAV photos not showing in Unsorted

### DISCUSSION THREAD

**Discussion by ChrisKibble** [photoprism/photoprism#3191] [General]
Hi Everyone,

I'm about a week into Photoprism and have been moving collections from various spots on to the platform and thought I had a knack for how the flow worked, but now that I'm doing it from my iPhone and using PhotoSync it's not quite going the same 100% of the time. With uploads, the photos always showed up in Unsorted, I would then select them and move them into their albums. Since I'm moving from other platforms that also use albums, I would just move them album by album.

With WebDAV, however, not all of the photos show up in unsorted. The photos are not missing - they import fine, and then go from my Import folder into my Originals folder.  They also show up in the Library if I manually go looking for them. But many of them skip right past Unsorted, so I never get an opportunity to put them into albums, which is how I'm organizing them. 

I've confirmed they open ok and that they are not stacked photos. They also don't appear under Hidden or Private, nor are there any errors in the logs. I also did a reindex on just the folder they're in, and then on the entire library. I brough down the container and brought it back up, and also restarted the server.

To confirm they weren't accidentally put into another album, I ran the following query:

```sql
select photos.id
     , photos.photo_name
     , albums.album_title
  from photos
  left join photos_albums
    on photos.photo_uid = photos_albums.photo_uid
  left join albums
    on photos_albums.album_uid = albums.album_uid
 where photos.photo_name = '20230210_031308_28B114D1' -- Missing from Unsorted and in no albums
    or photos.photo_name = '20170526_231506_592123A1' -- Shows up in Album
```

The results showed both the "good" and "bad" photo, with one having an album title and the other being `null`.

If I run the following query, I see 31 of my photos aren't in any album, but I'd expected this to be zero since I don't have any photos that are unsorted, are in private, or are in hidden. Unless I'm misunderstanding how it's supposed to function.

```sql
select photos.id
     , photos.photo_name
  from photos
  left join photos_albums
    on photos.photo_uid = photos_albums.photo_uid
 where photos_albums.album_uid is null
```

I'm running Photoprism on Ubuntu 22.10 with Docker 23.0.1. I have 4 GB of RAM and 4 GB of Swap. Database is MariaDB and I'm mostly out-of-the-box settings other than adding two additional users to Photoprism and that it's running behind nginx.

**Comment by lastzero** [comment]
Could it be this issue? https://github.com/photoprism/photoprism/issues/3051

**Reply by ChrisKibble** [reply]
Hi @lastzero, I don't *think* so since I've never deleted an album. If I switch to preview (just to see if any of those changes fix this), can I roll back to latest without impacting anything else?

**Reply by lastzero** [reply]
Yes. Note the preview isn't broken to the best of my knowledge, just can't do a full QA test and push a new stable release every day.

**Reply by ChrisKibble** [reply]
Ugh, this one's on me, @lastzero ... some mix of RTFM and PEBAC. Just found the actual review section that I've overlooked entirely (not realizing that "search" expanded out) and they're all in there. None of my transfers from other services ended up in there, but iOS uploads apparently found their way. Thank you for stepping in and trying to help, I *think* I've probably found them all. We'll see what it looks like after I organize all of these ones. (facepalm)
